mod/pages/pages/pages/history.php

```diff
@@ -20,23 +21,23 @@
 elgg_set_page_owner_guid($container->getGUID());
 
 if (elgg_instanceof($container, 'group')) {
-	elgg_push_breadcrumb($container->name, "pages/group/$container->guid/all");
+	elgg_push_breadcrumb(gc_explode_translation($container->name,$lang), "pages/group/$container->guid/all");
 } else {
-	elgg_push_breadcrumb($container->name, "pages/owner/$container->username");
+	elgg_push_breadcrumb(gc_explode_translation($container->name,$lang), "pages/owner/$container->username");
 }
 pages_prepare_parent_breadcrumbs($page);
-elgg_push_breadcrumb($page->title, $page->getURL());
+elgg_push_breadcrumb(gc_explode_translation($page->title,$lang), $page->getURL());
 elgg_push_breadcrumb(elgg_echo('pages:history'));
 
-$title = $page->title . ": " . elgg_echo('pages:history');
+$title = gc_explode_translation($page->title,$lang) . ": " . elgg_echo('pages:history');
 
 $content = elgg_list_annotations(array(
 	'guid' => $page_guid,
 	'annotation_name' => 'page',
 	'limit' => max(20, elgg_get_config('default_limit')),
 	'order_by' => "n_table.time_created desc",
 ));
-
+error_log($content);
```

Comment:
Looks all good. The only thing I would say is to remove this line to avoid clutter in the error log.

Reply:
Oups! Done @ethanWallace